### PR TITLE
[ticket/13716] Check phpBB version against config version

### PR DESCRIPTION
**Pull request by rmcgirr83**
Display a warning in the ACP if the version constant (files) does not match the phpbb_config version (often indicating that the database updates hasn't been run)

https://tracker.phpbb.com/browse/PHPBB3-13716

PHPBB3-13716

### CODE REVIEW

**Comment by RMcGirr83** [comment]
https://github.com/phpbb/phpbb/pull/3901

**Comment by RMcGirr83** [comment]
Lingo probably isn't the best, just tossed this together (basically copy paste) to get this ticket resolved.  No idea why the build isn't occurring though.

**Comment by RMcGirr83** [comment]
And this wouldn't have restarted because?

**Comment by RMcGirr83** [comment]
@marc1706 @Nicofuma @god

File: phpBB/language/en/install.php [hunk 1]
```diff
@@ -530,6 +533,28 @@
 	'UPDATE_DATABASE_SCHEMA'		=> 'Updating database schema',
 	'UPDATE_FILES'					=> 'Update files',
 	'UPDATE_FILES_NOTICE'			=> 'Please make sure you have updated your board files too, this file is only updating your database.',
+	'UPDATE_INCOMPLETE'				=> 'Your phpBB installation has not been correctly updated.',
+	'UPDATE_INCOMPLETE_MORE'		=> 'Please read the information below in order to fix this error.',
+	'UPDATE_INCOMPLETE_EXPLAIN'		=> '<h1>Incomplete update</h1>
```
Comment (Nicofuma) [inline phpBB/language/en/install.php]: I think it was decided to simplify this text.

@VSEphpbb

Comment (VSEphpbb) [inline phpBB/language/en/install.php]: Anything I had to say about this was said in the previous PR.

**Comment by RMcGirr83** [comment]
@VSEphpbb better?  Any suggestions?

**Comment by Nicofuma** [comment]
better, @VSEphpbb @CHItA any idea for the 3.2 version of the text?

**Comment by CHItA** [comment]
i don't see why would we need a separate patch for 3.2.

File: phpBB/language/en/install.php [hunk 2]
```diff
@@ -530,6 +533,12 @@
...
+
+		<p>We noticed that the last update of your phpBB installation hasn’t been completed. Visit the <a href="%1$s" title="%1$s">database_update script</a> and run it. If it is missing, please <a href="https://www.phpbb.com/downloads/" title="phpBB downloads">download your package version</a>, upload the "install" folder to your phpBB root directory (where your config.php file is) and <a href="%1$s" title="%1$s">run the database update script.</p>',
```
Comment (marc1706) [inline phpBB/language/en/install.php]: There is a missing ```</a>``` before the closing `</p>`.

File: phpBB/adm/style/acp_update.html [hunk 3]
```diff
@@ -38,6 +54,11 @@
 	</fieldset>
 <!-- END updates_available -->
 
+<!-- IF S_UPDATE_INCOMPLETE -->
+	{INCOMPLETE_INSTRUCTIONS}
+	<br><br>
```
Comment (marc1706) [inline phpBB/adm/style/acp_update.html]: Please remove one of the `<br>` tags. Looks better that way.

**Pull request merged**